apps/bbsync/query.py

```diff
@@ -220,11 +220,12 @@ def generate_keywords(self):
     def generate_flags(self):
         """
         generate query for Bugzilla flags
-        TODO: needinfo, nist_cvss_validation and other flags
+        TODO: needinfo and other flags
         """
         self._query["flags"] = []
         self.generate_hightouch_flags()
         self.generate_requires_doc_text_flag()
+        self.generate_nist_cvss_validation_flag()
```

Comment:
Nitpick: the order of these methods (lines 226-228 in this file and lines 831-833 in `convertors.py)` does not match the order of their implementation (lines 230-281 in this file). I would prefer to have it consistent for easier readability.

Reply:
I don't understand.

query.py:
```
226:        self.generate_hightouch_flags()
227:        self.generate_requires_doc_text_flag()
228:        self.generate_nist_cvss_validation_flag()
230:    def generate_hightouch_flags(self):
248:    def generate_requires_doc_text_flag(self):
264:    def generate_nist_cvss_validation_flag(self):
```
- generate_hightouch_flags() reads from flaw.major_incident_state
- generate_requires_doc_text_flag() reads from flaw.requires_summary
- generate_nist_cvss_validation_flag() reads from flaw.nist_cvss_validation

convertors.py:
```
831:            major_incident_state=self.major_incident_state,
832:            requires_summary=self.requires_summary,
833:            nist_cvss_validation=self.nist_cvss_validation,
```

Therefore all triplets of lines/blocks are in the same order:
- query.py 226 & query.py 230-246 & convertors.py 831
- query.py 227 & query.py 248-262 & convertors.py 832
- query.py 228 & query.py 264-280 & convertors.py 833

Please provide an actionable explanation of the code change you envision.

Reply:
Oh, I am sorry! I am looking at it again, and you are right. I made a mistake, sorry for the noise.